charts/istio-operator/files/gen-operator.yaml

```diff
@@ -175,7 +175,7 @@ spec:
       serviceAccountName: istio-operator
       containers:
         - name: istio-operator
-          image: istio/operator:1.15.5
+          image: gcr.io/istio-testing/operator:1.16-dev
```

Comment:
please do not use `dev` version. Is should be `1.16.4`

Reply:
I could not find the tag 1.16.4 on gcr.io
<img width="1294" alt="Screenshot 2023-04-24 at 16 00 57" src="https://user-images.githubusercontent.com/61050265/234026481-cd7554ea-98e5-4a66-8b9a-af8bf999697d.png">

but I tested by using the version tagged 1.16.4 on docker.io and it seems to work fine
<img width="575" alt="Screenshot 2023-04-24 at 16 05 29" src="https://user-images.githubusercontent.com/61050265/234026697-9bc4e585-3576-40e1-8610-4b489c27e935.png">

should we use that one?

Reply:
yes